components/CaseTable.tsx

```diff
@@ -36,7 +39,7 @@ export const CaseTable: React.FunctionComponent<ICaseTableProps> = (props) => {
     );
     const indexOfHtanParticipantId = _.findIndex(
         columns,
-        (c) => c.id === 'HTANParticipantID'
+        (c) => c.id === 'HTAN Participant ID'
```

Comment:
we have a string with spaces as ID?

Reply:
Right. We cannot use the name property to get the actual name of the column anymore, because name is now a component not a string. Ideally we should have another property like header to format the column header, but the actual table library uses name to render the header. So this is a workaround to use the id as name. Not a perfect solution though.